includes/KSES.php

```diff
@@ -105,6 +105,8 @@ public function filter_safe_style_css( $attr ) {
 	 * @see safecss_filter_attr()
 	 * @todo Use safe_style_disallowed_chars filter once WP 5.5+ is required.
 	 *
+	 * @SuppressWarnings(PHPMD)
```

Comment:
Suppress all warnings? Can you explain a little why this is happening here?

Reply:
Because this is mostly a copy from WordPress core.